compiler/rustc_symbol_mangling/src/typeid/typeid_itanium_cxx_abi.rs

```diff
@@ -913,12 +913,15 @@ fn transform_ty<'tcx>(tcx: TyCtxt<'tcx>, ty: Ty<'tcx>, options: TransformTyOptio
             );
         }
 
+        ty::Alias(..) => {
+            ty = tcx.normalize_erasing_regions(ty::ParamEnv::reveal_all(), ty);
```

Comment:
this type needs to be transformed as well, since it may normalize to, e.g., `*const c_void`

Reply:
Thanks for pointing it out! I'll fix it.